### ISSUE TITLE
core-size and fallback-size not taken into account

### ISSUE THREAD

**Issue by obouallou** [documents4j/documents4j#49]
Hello,

For the standalone-server, no matter what values I put for core-size and fallback-size, I always endup with 4 runners and 8 workers. (I have 4 cores CPU)
The values are correctly set in the StandaloneWebConverterConfiguration, but I think  this : 
```
ResourceConfig resourceConfig = ResourceConfig
                .forApplication(new WebConverterApplication(configuration))
                .register(configuration);
```
isn't behaving as expected.

Is it a bug ?

What version of grizzly were you using ? (running 2.6 here)

**Comment by raphw** [comment]
I am using Grizzly 2.6. 

I looked through the code and it seems like the pool size values are propagated correctly. What pool size did you check? Did you investigate if documents4j is sending the size values correctly?

**Comment by obouallou** [comment]
Here's a test I did with -S 10 -B 10:
https://ibb.co/dHM0rw

You can see, that the values are correction in builder, but the httpServer doesn't take them into account.

**Comment by raphw** [comment]
Yes, the server threads are controlled by Grizzly, theses sizes adjust the converters thread pool size.
